Integrations/DomainTools_Iris/iris.py

```diff
@@ -0,0 +1,393 @@
+from CommonServerPython import *
+import requests
+
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS '''
+
+BASE_URL = 'http://api.domaintools.com'
+USERNAME = demisto.params().get('username')
+API_KEY = demisto.params().get('apikey')
+RISK_THRESHOLD = int(demisto.params().get('risk_threshold'))
+PIVOT_COUNT_THRESHOLD = int(demisto.params().get('pivot_count_threshold'))
+YOUNG_DOMAIN_TIMEFRAME = int(demisto.params().get('young_domain_timeframe'))
+
+''' HELPER FUNCTIONS '''
+
+
+def http_request(method, path, other_params=None):
+    """
+    HTTP request helper function
+    """
+    params = {'app_partner': 'demisto', 'app_name': 'Iris Plugin', 'app_version': '1', 'api_username': USERNAME,
+              'api_key': API_KEY}
+    if other_params:
+        params.update(other_params)
+    url = '{}{}'.format(BASE_URL, path)
+    res = requests.request(
+        method=method,
+        url=url,
+        params=params
+    )
+    res_json = res.json()
+    if not res.ok:
+        txt = 'error in URL {} status code: {} reason: {}'.format(url, res.status_code,
+                                                                  res_json['error'].get('message'))
+        demisto.error(txt)
+        raise Exception(txt)
+
+    try:
+        return res_json.get('response')
+    except Exception as ex:
+        demisto.debug(str(ex))
+        demisto.results({"Type": entryTypes["error"], "ContentsFormat": formats["text"], "Contents": res.text})
+
+
+def find_age(create_date):
+    time_diff = datetime.now() - datetime.strptime(create_date, "%Y-%m-%d")
+    return time_diff.days
+
+
+def get_threat_component(components, threat_type):
+    for component in components:
+        if component['name'] == threat_type:
+            return component
+    else:
+        return None
+
+
+def convert_empty_to_null(data_obj):
```

Comment:
Not sure why we need this function. There's an argument `removeNull` of `tableToMarkdown`, try using it instead

Reply:
This cleans up the DomainTools context data. `removeNull` would not do what I am doing.